packages/buidler-core/src/internal/artifacts.ts

```diff
@@ -109,6 +111,22 @@ function isFullyQualified(name: string) {
   return name.includes(":");
 }
 
+export async function getAllArtifacts(
```

Comment:
Getting all artifacts is tricky because buildInfo files are also `.json`.

I'm pretty sure there must be a better way of doing this though.